src/elements/content-sidebar/activity-feed/activity-feed/ActivityThread.js

```diff
@@ -0,0 +1,17 @@
+// @flow
+import * as React from 'react';
+import ActivityCard from '../ActivityCard';
+
+type Props = {
+    children: React.Node,
+    hasReplies: boolean,
+};
+
+const ActivityThread = ({ children, hasReplies }: Props) => {
+    if (hasReplies) {
+        return <ActivityCard className="bcs-ActivityThread">{children}</ActivityCard>;
```

Comment:
It looks like this will create a nested ActivityCard output, since the `Comment` component's root is also ActivityCard. Is that desirable? It also doesn't look like `.bcs-ActivityThread` is used at the moment. Will it be in the future?

Reply:
I am aware of it, but I think it would be best to treat ActivityThread as ActivityCard containing main comment with replies and wrapping it up. I think it shouldn't be a problem to have nested ActivityCard for parent comment and each reply. Probably it would just need style fixes for the spacing for the parent component but IMO it would look like more consistent.

Reply:
And I would keep the className - it will be used in next [PR](https://github.com/box/box-ui-elements/pull/3010/files)